From 876c1c4a7f66346ecd2295c77009553d23debd63 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:36:13 -0400
Subject: [ethernet] i40e: remove unnecessary delay

Message-id: <1409658042-22689-13-git-send-email-sassmann@redhat.com>
Patchwork-id: 90208
O-Subject: [RHEL7.1 PATCH 012/281] i40e: remove unnecessary delay
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

Ain't nothing gonna break my stride, nobody's gonna slow me down,
oh no. I got to keep on moving.

This was originally put in for debugging just-in-case purposes
and never removed.

Change-ID: Ic12c2e179c3923f54e6ba0a9e4ab05d25c3bab29
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 81b8c0117d88adb9c72b3fa5feff0d1efac816bd)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c b/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
index 7d133fa..189e250 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
@@ -671,7 +671,6 @@ void i40e_reset_vf(struct i40e_vf *vf, bool flr)
 complete_reset:
  /* reallocate vf resources to reset the VSI state */
  i40e_free_vf_res(vf);
- mdelay(10);
  i40e_alloc_vf_res(vf);
  i40e_enable_vf_mappings(vf);
  set_bit(I40E_VF_STAT_ACTIVE, &vf->vf_states);
-- 
1.7.1